src/shipit_static_analysis/shipit_static_analysis/config.yml

```diff
@@ -2,19 +2,36 @@ target: obj-x86_64-pc-linux-gnu
 
 # All the clang checks used by the bot
 clang_checkers:
- - -*
- - clang-analyzer-deadcode.DeadStores
- - modernize-loop-convert
- - modernize-raw-string-literal
 # modernize-use-auto (controversial, see bug 1371052)
 # modernize-use-bool-literals (too noisy because of `while (0)` in many macros)
- - modernize-use-equals-default
- - modernize-use-equals-delete
- - modernize-use-nullptr
- - modernize-use-override
- - mozilla-*
- - performance-faster-string-find
- - performance-for-range-copy
- - readability-else-after-return
- - readability-misleading-indentation
- - readability-redundant-control-flow
+ - name: -*
+   publish: no
+ - name: clang-analyzer-deadcode.DeadStores
+   publish: no
+ - name: modernize-loop-convert
```

Comment:
Could you please add a few checkers from https://github.com/mozilla-releng/services/pull/525 ? I.e.:

- [ ] clang-analyzer-security.*
- [ ] misc-assert-side-effect
- [ ] misc-suspicious-missing-comma
- [ ] misc-suspicious-semicolon
- [ ] modernize-redundant-void-arg
- [ ] modernize-shrink-to-fit
- [ ] readability-redundant-smartptr-get
- [ ] readability-uniqueptr-delete-release